staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/helpers.go

```diff
@@ -129,6 +129,16 @@ func HasServedCRDVersion(crd *CustomResourceDefinition, version string) bool {
 	return false
 }
 
+// HasServedCRDVersionInSpec returns true if the given version is in the list of CRD's versions and the Served flag is set.
+func HasServedCRDVersionInSpec(spec *CustomResourceDefinitionSpec, version string) bool {
```

Comment:
not sure what's the best way to do it cleanly. Perhaps we could put a private method in apiserver package to avoid the api-change label

nit: there are two methods in this file do the same thing: `HasServedCRDVersion` and `HasVersionServed`

Reply:
I'd probably start with a private method to limit size, especially if we intend to pick this change back. As a follow-up, I'd collapse the two existing methods to a single one that takes a CustomResourceDefinitionSpec as a param